JustSaying/JustSayingFluentlyExtensions.cs

```diff
@@ -1,11 +1,44 @@
 using System;
+using JustSaying.AwsTools;
+using JustSaying.AwsTools.QueueCreation;
 using JustSaying.Messaging.MessageHandling;
+using JustSaying.Messaging.MessageSerialisation;
+using JustSaying.Messaging.Monitoring;
 using JustSaying.Models;
 
 namespace JustSaying
 {
     public static class JustSayingFluentlyExtensions
     {
+        public static IMayWantOptionalSettings InRegion(this JustSayingFluentlyLogging logging, string region) => InRegions(logging, region);
+
+        public static IMayWantOptionalSettings InRegions(this JustSayingFluentlyLogging logging, params string[] regions)
```

Comment:
We could do, I'm not sure it adds much though, I presume it's done like this to enable `...InRegions("uk","fr")` as well as `...InRegions(tenants.ToArray())`.
Yeah, let me do that unless anyone disagrees.